lib/micro/case.rb

```diff
@@ -239,6 +239,19 @@ def Failure(type = :error, result: nil)
         __get_result(false, value, type)
       end
 
+
+      def Try(type = nil, catch: StandardError, on: {})
```

Comment:
@MatheusRich follows a suggestion based on this feedback: https://github.com/serradura/u-case/pull/87#pullrequestreview-565887654. 

```suggestion
      def Try(type = nil, catch: StandardError, on: Kind::Empty::HASH)
```